GitDepend/Busi/Nuget.cs

```diff
@@ -62,10 +62,17 @@ private ReturnCode ExecuteNuGetCommand(string arguments)
                 .ToArray();
 
             var code = NuGet.CommandLine.Program.Main(args);
-
+            var output = sb.ToString();
+            var hasWarnings = output.ToLower().Contains("warning");
+            
             Console.SetOut(oldOut);
             
             _console.WriteLine($"nuget {arguments}");
+            if (hasWarnings)
+            {
+                _console.WriteLine(output);
+                code = 3;
```

Comment:
After looking more, you are not returning the 3. But, it's still not clear what the magic number means.